File: answers/C/@mykeels/10-hex-color-generator.c
#include <stdio.h>
#include <time.h>
#include <stdlib.h>

int main() {
    srand(time(NULL));
    for (int i = 1; i <= 6; i++) {
        int c = rand() % 16;
        if (c < 10) {
            printf("%d", c);
        }
        else {
            if (c == 10) {
                printf("a");
            }
            else if (c == 11) {
                printf("b");
            }
            else if (c == 12) {
                printf("c");
            }
            else if (c == 13) {
                printf("d");
            }
            else if (c == 14) {
                printf("e");
            }
            else if (c == 15) {
                printf("f");
            }
        }
    }
    printf("\n");
    return 0;
}

File: answers/Go/@codehakase/10-hex-color-generator.go
package main

import (
	"fmt"
	"math/rand"
	"strconv"
	"time"
)

func main() {
	rand.Seed(time.Now().UnixNano())

	fmt.Printf("#%s%s%s\n",
		strconv.FormatInt(int64(rand.Intn(256)), 16),
		strconv.FormatInt(int64(rand.Intn(256)), 16),
		strconv.FormatInt(int64(rand.Intn(256)), 16))
}
